### PR TITLE
Add support for travis ci

### PR DESCRIPTION
**Pull request by morihladko**
I created travis account and added travis config. I had to make some minor changes in setup.py and had to rename test packages to follow a pattern.

Travis should build on every pull request and on every commit in _master_ branch. It will build and upload the package into _pypi_ on every tagged commit. 

I also propose to tag with the pattern _x.x.x_ without the _v_ prefix, _pallets_ (_flask_, _click_) and _django_ don't have it either (but _celery_ and _requests_ do).

### CODE REVIEW

**Comment by babky-atteq** [comment]
certainly do not use v in tagging.

File: requirements.txt [hunk 1]
```diff
@@ -1,15 +1,12 @@
 future
 zsl_client
```
Comment (babky-atteq) [inline requirements.txt]: Do we really need zsl_client in requirements for zsl?

File: zsl/resource/model_resource.py [hunk 2]
```diff
@@ -93,6 +93,7 @@ def __init__(self, params, args, data):
         self._data = data
 
         # test if params is a list
+        # TODO: replace this with a better is_list test
```
Comment (babky-atteq) [inline zsl/resource/model_resource.py]: Why a TODO? make it a is_list method and that's all.

File: requirements.txt (hunk 1 above)
Comment (morihladko) [inline requirements.txt]: Yes, there's is a background_task decorator to add a task into job queue, which sends it with help of zsl_client. I think it is not used anywhere and was build as a proof of concept, but it is still there.

Comment (babky-atteq) [inline requirements.txt]: Ok, cool.

**Pull request merged**